internal/pkg/agent/cmd/install.go

```diff
@@ -40,7 +42,7 @@ would like the Agent to operate.
 
 	cmd.Flags().BoolP("force", "f", false, "Force overwrite the current and do not prompt for confirmation")
 	cmd.Flags().BoolP("non-interactive", "n", false, "Install Elastic Agent in non-interactive mode which will not prompt on missing parameters but fails instead.")
-	cmd.Flags().String("base-path", paths.DefaultBasePath, "Base path for installing Elastic Agent's files.")
+	cmd.Flags().String(flagInstallBasePath, paths.DefaultBasePath, "Base path (absolute) for installing Elastic Agent's files.")
```

Comment:
[Suggestion]
I think we could take the opportunity and make it clearer.

```suggestion
	cmd.Flags().String(flagInstallBasePath, paths.DefaultBasePath, "The path where the Elastic Agent will be installed. It must be an absolute path.")
```